py/torch_tensorrt/dynamo/conversion/impl/split.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
+
+import numpy as np
+import torch
+import torch_tensorrt as trt
+from torch import Tensor
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion.impl.shape import get_shape_with_dynamic_shape
+from torch_tensorrt.fx.converters.converter_utils import (
+    has_dynamic_shape,
+    set_layer_name,
+)
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+
+def split(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    split_size_or_sections: Union[int, List(int)],
+    dim: Optional[Any] = 0,
```

Comment:
Replace with `dim: int = 0`, since this is a mandatory argument, as per [`native_functions.yaml`](https://github.com/pytorch/pytorch/blob/22f5889753d38661aefeb8ea7110ff5787a1b5e9/aten/src/ATen/native/native_functions.yaml#L5288-L5299).